src/Symfony/Bundle/WebProfilerBundle/Resources/views/Collector/request.html.twig

```diff
@@ -18,11 +18,31 @@
         {% endif %}
     {% endset %}
 
+    {% if collector.forward %}
```

Comment:
What do you think about creating a simple Twig macro to avoid repeating in `forward_handler` the same logic as the previous `request_handler`?